include/SDL3/SDL_stdinc.h

```diff
@@ -1291,6 +1291,22 @@ extern SDL_DECLSPEC int SDLCALL SDL_strcasecmp(const char *str1, const char *str
  */
 extern SDL_DECLSPEC int SDLCALL SDL_strncasecmp(const char *str1, const char *str2, size_t maxlen);
 
+
+
+/**
+ * Locate the first occurrence of any character in the accept string.
```

Comment:
It took me a couple reads before understanding; is this a standard description? Can we snatch [cppreference.com's definition](https://en.cppreference.com/w/c/string/byte/strpbrk) or use something similar?

Reply:
The SDL_strpbrk and SDL_RESTRICT can be removed from the series.
I intended to use it for preparing the arguments of `CreateProcess`, but ended up not using it.

Reply:
I wrote this in a hush, so I did not pay too much attention to the docs (English is not my mother tongue)